### PR TITLE
refactor: extract form styles to theme

### PR DESCRIPTION
**Pull request by mrfelton**
## Description:

Extracts form input styles to the theme and refactors form components to use new `InputField` component in order to standardize sub component layout

## Motivation and Context:

Ability to style something as an Input field outside of the context of Informed.

## How Has This Been Tested?

- Verify forms look correct in the app
- Verify InputField component in the storybook

## Types of changes:

Refactor

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

File: renderer/components/Form/Input.js [hunk 1]
```diff
@@ -62,77 +54,77 @@ const Input = ({
     return !maskedValue && maskedValue !== 0 ? '' : maskedValue
   }
 
+  const fieldDescription = type !== 'hidden' && description
+  const fieldLabel = type !== 'hidden' && label
+  const fieldError =
+    type !== 'hidden' && hasMessage && (error || asyncError) && (error || asyncError)
```
Comment (korhaliv) [inline renderer/components/Form/Input.js]: `(error || asyncError)` two times?

File: renderer/components/Form/Input.js [hunk 2]
```diff
@@ -62,77 +54,77 @@ const Input = ({
     return !maskedValue && maskedValue !== 0 ? '' : maskedValue
   }
 
+  const fieldDescription = type !== 'hidden' && description
```
Comment (korhaliv) [inline renderer/components/Form/Input.js]: type !== 'hidden' can be extracted to improve readability a bit

File: renderer/components/Form/TextArea.js [hunk 3]
```diff
@@ -47,20 +48,24 @@ const TextArea = ({
     }
     return !maskedValue && maskedValue !== 0 ? '' : maskedValue
   }
+  const fieldError = hasMessage && (error || asyncError) && (error || asyncError)
```
Comment (korhaliv) [inline renderer/components/Form/TextArea.js]: `(error || asyncError)` two times?

**Comment by mrfelton** [comment]
Updated

**Pull request merged**